terraform/cloudrun.tf

```diff
@@ -57,6 +57,17 @@ resource "google_cloud_run_v2_service" "metrics" {
   }
 }
 
+resource "google_cloud_run_v2_service_iam_binding" "public_metrics_access" {
```

Comment:
Please use the `_iam_member` resource instead of `_iam_binding`

Reply:
Done